Ubuntu.Dev.Compose.yaml

```diff
@@ -0,0 +1,27 @@
+services:
+  wireguard-dashboard:
+    build:
+      context: .
+      dockerfile: Ubuntu.Dev.Dockerfile
+    #image: donaldzou/wgdashboard:latest
+    restart: unless-stopped
+    container_name: wgdashboard
+      #environment:
```

Comment:
Environment can be removed as  `entrypoint.sh` script from the current Docker image is ignored.